src/engine/strat_engine/blockdev.rs

```diff
@@ -7,7 +7,7 @@
 use std::fs::OpenOptions;
 use std::path::PathBuf;
 
-use time::Timespec;
+use chrono::{DateTime, Local};
```

Comment:
Before the change, the code in `BlockDevMgr::save_state` was using `time::now()`, which returns the local time, so I assumed from there that the local time was a requirement. 

I don't know all the stratisd details and cannot make that decision. But local time seems to make more sense to me. Should UTC be preferred, I can easily make that change.